src/navsections/K8sResourceSectionBlueprint/ResourceKindSectionNameCounter.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {useMemo} from 'react';
+
+import {SectionCustomComponentProps} from '@models/navigator';
+
+import {useAppSelector} from '@redux/hooks';
+
+import {countResourceErrors, countResourceWarnings} from '@utils/resources';
+
+import * as S from '../../components/molecules/SectionRenderer/styled';
+import * as WarningStyle from '../../components/organisms/NavigatorPane/WarningAndErrorsDisplay.styled';
```

Comment:
I do not like this too much. Suggestions on how you share style within this project?

Reply:
I would say just duplicate the styles that you need for now. Or you could create a `components` folder under `src/styles` and move the reusable styles there.

I think so far in the code-base we duplicated the needed styles just to be able to change their styles separately if we did big redesigns.

Reply:
Or, your idea to extract these as Atoms would be good as well, but only if they have some kind of logic.. if the component is strictly a styled component with no logic, maybe having them in the `styles` folder is better?!

Any thoughts @andreiv1992 @erdkse @jmelnych ?

Reply:
As for sharable styles - we can have all app shared styles in App.styled.ts file, or create a folder like @devcatalin suggested. As for components atoms - I think that components imply to have some render logic. If it is solely about styles - let's keep that in styles, this is my suggestion.

Reply:
I decided to go for duplicated style, neither extraction felt right.
1/ The icon in navigator does not have the "selected" logic.
2/ The icon in a single resource does not include a count.

The current look is this

<img width="244" alt="Screenshot 2022-04-07 at 10 04 07" src="https://user-images.githubusercontent.com/7761005/162151474-68ea6498-8750-4711-bc09-5688b5980db4.png">

Reply:
Looks good to me!